Documentation/demo.md

```diff
@@ -271,7 +271,10 @@ etcdctl --endpoints=$ENDPOINTS endpoint health
 
 <img src="https://storage.googleapis.com/etcd/demo/11_etcdctl_snapshot_2016051001.gif" alt="11_etcdctl_snapshot_2016051001"/>
 
+Snashot can only be requested from one etcd node, so `--snapshot` flag should contain only one endpoint.
```

Comment:
Typo :) - Snashot vs `Snapshot`  @vimalk78

Reply:
Thanks @spzala , updated.

Reply:
@vimalk78 thanks, nice job in concisely taking care of the issue.